src/institutions/container.jsx

```diff
@@ -1,12 +1,26 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import fetchInstitutions from '../actions/fetchInstitutions.js'
+import requestInstitutions from '../actions/requestInstitutions.js'
+import fetchEachInstitution from '../actions/fetchEachInstitution.js'
+import receiveInstitutions from '../actions/receiveInstitutions.js'
 import Institutions from './index.jsx'
+import { getKeycloak } from '../utils/keycloak.js'
 
 export class InstitutionContainer extends Component {
   componentDidMount() {
     if (!this.props.institutions.fetched && !this.props.institutions.isFetching)
-      this.props.dispatch(fetchInstitutions())
+      this.props.dispatch(requestInstitutions())
+
+    const leis = getKeycloak().tokenParsed.lei.split(',')
```

Comment:
This, and the few lines below, are now how we fetch the institutions ...
- get the `lei`s from the token
- build the array of objects
- call `fetchEachInstitution` with that array